packages/@sanity/desk-tool/src/tool/DeskToolPanes.js

```diff
@@ -249,6 +249,7 @@ export default class DeskToolPanes extends React.Component {
                     isCollapsed={isCollapsed}
                     isSelected={i === panes.length - 1}
                     isClosable={siblingIndex > 0}
+                    isActive={i === panes.length - 2}
```

Comment:
what's the significance of second to last for `isActive`?

Reply:
The second-to-latest pane contains the currently active selection (the bright blue selection, as opposed to the inactive selection in the panes where index < panes.length - 2):
![image](https://user-images.githubusercontent.com/406933/128731724-07e22c20-bd7b-4446-b029-fedac7f4f5d2.png)